src/lib/y2network/sysconfig/dns_reader.rb

```diff
@@ -31,9 +33,11 @@ class DNSReader
       #
       # @return [Y2Network::DnsConfig] DNS configuration
       def config
+        installer = Yast::Mode.installation || Yast::Mode.autoinst
```

Comment:
what about autoinst mode in second stage?

Reply:
network configuration runs completely in first stage. So in my POV when something invokes it in second stage then it should behave as on installed system.